controllers/comments.controllers.js

```diff
@@ -0,0 +1,15 @@
+const { selectCommentsForArticle } = require("../models/comments.models");
+
+exports.getCommentsForArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  selectCommentsForArticle(article_id)
+    .then((comments) => {
+      if(comments.length === 0){
```

Comment:
this functionality for errors should be in the model if its used. However for this we want a 200 if an article exists but doesnt have any comments yet!